File: app/src/main/java/com/example/aamezencev/weatherinfo/data/owmApi/JsonWindInfo.java
package com.example.aamezencev.weatherinfo.data.owmApi;

import com.google.gson.annotations.SerializedName;

/**
 * Created by aa.mezencev on 18.01.2018.
 */

public class JsonWindInfo {
    @SerializedName("speed")
    private String speed;
    @SerializedName("deg")
    private String deg;

    public JsonWindInfo(String speed, String deg) {
        this.speed = speed;
        this.deg = deg;
    }

    public String getSpeed() {
        return speed;
    }

    public String getDeg() {
        return deg;
    }
}

File: app/src/main/java/com/example/aamezencev/weatherinfo/data/owmApi/JsonWindInfo.kt
package com.example.aamezencev.weatherinfo.data.owmApi

import com.google.gson.annotations.SerializedName

/**
 * Created by aa.mezencev on 18.01.2018.
 */

data class JsonWindInfo(@field:SerializedName("speed") val speed: String,
                        @field:SerializedName("deg") val deg: String)
